### PR TITLE
Update administrative-templates-import-custom.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps on how to identify a dependency 

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3331

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/configuration/administrative-templates-import-custom.md [hunk 1]
```diff
@@ -67,6 +67,16 @@ This article shows you how to import custom ADMX and ADML files in the Endpoint
 
   To remove a dependency prerequisite, delete the associated ADMX file first. Then, delete the dependency prerequisite. In our Mozilla Firefox example, delete `firefox.admx` and then delete `mozilla.admx`.
   
+  To see if your ADMX has a dependency, open it in a text editor and look for using references in the policyNamespaces node. For example, this `kerberos.admx` file requires the `Windows.admx` file be uploaded to Intune first
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/administrative-templates-import-custom.md:70]:
```suggestion
  To see if your ADMX has a dependency, open it in a text editor and look for `using references` in the `policyNamespaces` node. For example, this `kerberos.admx` file requires the `Windows.admx` file to be uploaded to Intune first:
```

File: memdocs/intune/configuration/administrative-templates-import-custom.md [hunk 2]
```diff
@@ -67,6 +67,16 @@ This article shows you how to import custom ADMX and ADML files in the Endpoint
...
+  
+```Syntax
+ <policyNamespaces>
+    <target prefix="kerberos" namespace="Microsoft.Policies.Kerberos" />
+    <using prefix="windows" namespace="Microsoft.Policies.Windows" />
+  </policyNamespaces>
+```
+
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/administrative-templates-import-custom.md:79]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**